kv/tikv/raftstore/db_writer.go

```diff
@@ -165,13 +163,13 @@ func (writer *raftDBWriter) Close() {
 // 	return writer.checkResponse(cb.resp, len(b.requests))
 // }
 
-type RaftError struct {
-	RequestErr *errorpb.Error
-}
+// type RaftError struct {
+// 	RequestErr *errorpb.Error
+// }
 
-func (re *RaftError) Error() string {
-	return re.RequestErr.String()
-}
+// func (re *RaftError) Error() string {
+// 	return re.RequestErr.String()
+// }
```

Comment:
Just delete these rather than commenting them out - they will be in the Git history

Reply:
The reason why I comment out the code instead of deletion is to keep the code that I can refer to. All the git history will be squashed once this project is finished.